domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -46,6 +47,20 @@ fun <T : Path> _endsWith(expect: Expect<T>, expected: Path): Assertion =
 fun <T : Path> _endsNotWith(expect: Expect<T>, expected: Path) =
     ExpectImpl.builder.createDescriptive(expect, ENDS_NOT_WITH, expected) { !it.endsWith(expected) }
 
+fun <T : Path> _hasSameTextualContentAs(expect: Expect<T>, targetPath: Path, sourceCharset: Charset, targetCharset: Charset) =
+    ExpectImpl.builder.createDescriptive(expect, HAS_SAME_TEXTUAL_CONTENT, targetPath) {
+        val sourceFileText = it.toFile().readText(sourceCharset)
+        val targetFileText = targetPath.toFile().readText(targetCharset)
+        return@createDescriptive sourceFileText == targetFileText
+    }
```

Comment:
add `import ch.tutteli.niok.readText` and then rewrite it to
```suggestion
    ExpectImpl.builder.createDescriptive(expect, HAS_SAME_TEXTUAL_CONTENT, targetPath) {
        it.readText(sourceCharset) == targetPath.readText(targetCharset)
    }
```

Reply:
Moreover, I was asking myself if we should include the chosen encodings in reporting if one (or both) differ from UTF-8. What do you think?

Reply:
I think it might be useful for user